workloads/oai/pkg-example-cucp-bp/apply-replacements-owner.yaml

```diff
@@ -24,20 +24,6 @@ replacements:
     options:
       delimiter: '-'
```

Comment:
I have found apply-replacements with delimiters to be very fragile, I would avoid them if possible (maybe a future update, or eventually I would hope these become unnecessary with the implementation of CEL for function inputs)

Reply:
Sure @johnbelamaric , we will consider it in the next updates.